packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -17,45 +17,67 @@ interface IResponseException extends Error {
 	message: string;
 	code: number;
 	stack?: string;
-	underlyingResponseHeaders?: { [key: string]: any };
+	underlyingResponseHeaders?: { [key: string]: unknown };
 }
 
 /**
+ * Type guard for determining if an error is an {@link IResponseException }
  * @internal
  */
-export function exceptionToResponse(err: any): IResponse {
+function isResponseException(err: unknown): err is IResponseException {
+	return (
+		err !== null &&
+		typeof err === "object" &&
+		"errorFromRequestFluidObject" in err &&
+		(err as { errorFromRequestFluidObject: unknown }).errorFromRequestFluidObject === true
+	);
+}
+
+/**
+ * Converts an error object into an {@link @fluidframework/core-interfaces#IResponse}
+ * @internal
+ */
+export function exceptionToResponse(err: unknown): IResponse {
 	const status = 500;
-	if (err !== null && typeof err === "object" && err.errorFromRequestFluidObject === true) {
-		const responseErr: IResponseException = err;
+	if (isResponseException(err)) {
```

Comment:
Nit (while we're here): can we call this `error`? That would be required if we upgrade the config to "strict" anyways.